client/src/Dashboard/DashboardTitle.js

```diff
@@ -25,10 +26,35 @@ export default function DashboardTitle({ dates, userState, getDashboardData }) {
     }
   }, [dates, dateFilterValue])
 
+  useEffect(() => {
+    const mediaQueryList = window.matchMedia('(max-width: 560px)')
+    // addEventListener is unavailable for media query lists in Safari 13 or below
+    const isSafari13 = typeof mediaQueryList.addEventListener !== 'function'
+    const handleWidthChange = evt => setIsSmallDevice(evt.matches)
+
+    handleWidthChange(mediaQueryList)
+
+    if (isSafari13) {
+      mediaQueryList.addListener(handleWidthChange)
+    } else {
+      mediaQueryList.addEventListener('change', handleWidthChange)
+    }
+
+    return () => {
+      if (isSafari13) {
+        mediaQueryList.removeListener(handleWidthChange)
+      } else {
+        mediaQueryList.removeEventListener('change', handleWidthChange)
+      }
+    }
+  }, [])
+
   return (
     <div className="dashboard-title m-2">
-      <div className="flex items-center mb-3">
-        <Typography.Title className="dashboard-title mr-4">
+      <div className={`flex items-center mb-3 ${isSmallDevice && 'flex-col'}`}>
```

Comment:
for instance: you could add the necessary breakpoints (e.g., `sm`, `md`, etc.) to these tailwind classes as needed instead of the isSmallDevice and useEffect logic. 

What do you think?

Reply:
@cjhaddad, thanks for the link. I had no idea I could do this with Tailwind. I'll make changes.